homeassistant/components/vizio/config_flow.py

```diff
@@ -63,7 +86,7 @@ def _get_pairing_schema(input_dict: Dict[str, Any] = None) -> vol.Schema:
 
     return vol.Schema(
         {vol.Required(CONF_PIN, default=input_dict.get(CONF_PIN, "")): str},
-        extra=vol.ALLOW_EXTRA,
+        extra=vol.REMOVE_EXTRA,
```

Comment:
Why remove extra and not use the default, which is to fail on any extra keys?

Reply:
That is better, I changed it because I realized ALLOW_EXTRA did not make sense and did not realize `PREVENT_EXTRA` existed and was default